libcxx/include/module.modulemap

```diff
@@ -1,2111 +1,1609 @@
 
-module std_private_exception_exception        [system] { header "__exception/exception.h" }
-module std_private_exception_exception_ptr    [system] {
-  header "__exception/exception_ptr.h"
-  export std_private_exception_operations
-}
-module std_private_exception_nested_exception [system] { header "__exception/nested_exception.h" }
-module std_private_exception_operations       [system] { header "__exception/operations.h" }
-module std_private_exception_terminate        [system] { header "__exception/terminate.h" }
-
-module std_private_expected_bad_expected_access [system] { header "__expected/bad_expected_access.h" }
-module std_private_expected_expected            [system] { header "__expected/expected.h" }
-module std_private_expected_unexpect            [system] { header "__expected/unexpect.h" }
-module std_private_expected_unexpected          [system] { header "__expected/unexpected.h" }
-
-module std_private_format_buffer                          [system] { header "__format/buffer.h" }
-module std_private_format_concepts                        [system] { header "__format/concepts.h" }
-module std_private_format_container_adaptor               [system] { header "__format/container_adaptor.h" }
-module std_private_format_enable_insertable               [system] { header "__format/enable_insertable.h" }
-module std_private_format_escaped_output_table            [system] { header "__format/escaped_output_table.h" }
-module std_private_format_extended_grapheme_cluster_table [system] { header "__format/extended_grapheme_cluster_table.h" }
-module std_private_format_format_arg                      [system] { header "__format/format_arg.h" }
-module std_private_format_format_arg_store                [system] { header "__format/format_arg_store.h" }
-module std_private_format_format_args                     [system] { header "__format/format_args.h" }
-module std_private_format_format_context                  [system] {
-  header "__format/format_context.h"
+  module pstl {
+    header "__pstl/backend.h"
+    header "__pstl/backend_fwd.h"
+    header "__pstl/backends/default.h"
+    header "__pstl/backends/libdispatch.h"
+    header "__pstl/backends/serial.h"
+    header "__pstl/backends/std_thread.h"
+    header "__pstl/cpu_algos/any_of.h"
+    header "__pstl/cpu_algos/cpu_traits.h"
+    header "__pstl/cpu_algos/fill.h"
+    header "__pstl/cpu_algos/find_if.h"
+    header "__pstl/cpu_algos/for_each.h"
+    header "__pstl/cpu_algos/merge.h"
+    header "__pstl/cpu_algos/stable_sort.h"
+    header "__pstl/cpu_algos/transform.h"
+    header "__pstl/cpu_algos/transform_reduce.h"
+    header "__pstl/dispatch.h"
+    header "__pstl/handle_exception.h"
+    export *
+  }
+
+  module miscellaneous {
+    header "__bit_reference"
+    header "__fwd/bit_reference.h"
+    header "__hash_table"
+    header "__node_handle"
+    header "__split_buffer"
+    header "__std_mbstate_t.h"
+    header "__tree"
+    header "__verbose_abort"
+    export *
+  }
```

Comment:
I think we should just make `std` level modules for these individually. There doesn't seem much reason to put them together.